src/Microsoft.OData.Edm/Csdl/Semantics/CsdlSemanticsVocabularyAnnotation.cs

```diff
@@ -127,15 +136,20 @@ protected IEdmTerm ComputeTerm()
         private IEdmExpression ComputeValue()
         {
             IEdmTypeReference termType = Term is UnresolvedVocabularyTerm ? null : Term.Type;
-            CsdlExpressionBase adjustedExpression = AdjustStringConstantUsingTermType((this.Annotation).Expression, termType);
+            // Note: Attempted to create a wrapped version for the term's default value, but could not access said value.
+            /*if (!this.Annotation.UsesDefault)
+            {
+                var defaultExpression = new CsdlSemanticsStringConstantExpression(Term.DefaultValue, this.Schema);
+                return CsdlSemanticsModel.WrapExpression(defaultExpression, TargetBindingContext, this.Schema);
+            }*/

```

Comment:
These seems the appropriate place to put the logic to compute the default value. Was the issue that the term was not found, or that Term.DefaultValue had not been set?

Reply:
I had thought that I hadn't been able to access the default value, just the term, but it seems to be fine now. It's now working so that a CSDL annotation's value is the default value when applicable!